src/high-level/specialize-tensor.lisp

```diff
@@ -0,0 +1,31 @@
+;;;; specialize-tensor.lisp
+;;;;
+;;;; Author: Cole Scott
+
+(in-package #:magicl)
+
+;; XXX: This is a little overkill for the small amount of compatible
```

Comment:
This comment kinda summarizes this. It is definitely overkill for now but it will help when we want to start supporting broadcasting matrices